src/main/java/com/polishuchenko/bookstore/model/Category.java

```diff
@@ -0,0 +1,23 @@
+package com.polishuchenko.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "category")
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false)
+    private String name;
+    private String description;
+    @Column(nullable = false)
+    private boolean isDeleted = false;
```

Comment:
You added this column for soft delete but did not add annotation to actually change status instead of deleting. Or did I miss that somewhere?